roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -77,6 +82,21 @@
     resource_name: rolebinding-reader
     user: "system:serviceaccount:{{ openshift_logging_elasticsearch_namespace }}:aggregated-logging-elasticsearch"
 
+# needed for prometheus.  can remove other bindings?
+- oc_adm_policy_user:
+    state: present
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    resource_kind: cluster-role
+    resource_name: cluster-reader
```

Comment:
is there anything more specific we should be using here?  The requirement is to be able to create the token review requests

@enj

Reply:
I do not understand why this is needed at all.  Who is making token review requests (show me the code that does it).

Reply:
this goes along with the SA we were discussing yesterday.  You configure oauth-proxy to use an SA, that SA is used to create the request.  That code isn't anything new that Integration Services introduced.

Reply:
Comfirmed this change is required to be able to make it through the proxy to read metrics.